File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ""
    for i in s:
        if i == " ":
            answer += i
        
        elif i.isupper():
            print(ord(i), ord('A'))
            answer += chr((ord(i) - ord('A') + n) % 26 + ord('A'))
        elif i.islower():
            print(ord(i), ord('a'))
            answer += chr((ord(i) - ord('a') + n) % 26 + ord('a'))

    return answer

File: 프로그래머스/1/12926. 시저 암호/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        int asciiValue = 0;
        
        for(char c : s.toCharArray()) {
            if (c == ' ') {
                answer += c;
            }
            else if(Character.isUpperCase(c)) {
                asciiValue = ((int) c - (int) 'A' + n) % 26 + (int) 'A';
                answer += (char) asciiValue;
            }
            else if (Character.isLowerCase(c)) {
                asciiValue = ((int) c - (int) 'a' + n) % 26 + (int) 'a';
                answer += (char) asciiValue;
            }
        }
        
        return answer;
    }
}